src/main/java/me/gallowsdove/foxymachines/Items.java

```diff
@@ -422,6 +422,7 @@ public final class Items{
             Material.NETHERITE_SWORD,
             "&cCursed Sword",
             "&7Life Steal I",
+            "&7Armor Ignorance I",
```

Comment:
Maybe armor penetration would be better? Just opinion, don't have to change it if you like this ^^

Reply:
Ooooh I like that better yeah